### DISCUSSION TITLE
Patching Gaps in the CIS Windows 11 Benchmark - Level 1 Windows 11 - odds+endpoints

### DISCUSSION THREAD

**Discussion by giscus** [ennnbeee/oddsandendpoints-web#5] [Comments]
# Patching Gaps in the CIS Windows 11 Benchmark - Level 1 Windows 11

Updating the Center for Internet Security (CIS) benchmark build kit for Windows 11 in Microsoft Intune for level 1 security settings.

https://memv.ennbee.uk/posts/windows-cis-patching-gaps-part3/

**Comment by alevalesantos** [comment]
Excellent article.
Right now I'm facing these problems in implementing CIS and you helped a lot in fix the issues.
I'm using OMA-URI (CSP) to implement the CIS baseline but the SID settings didn't work, 
Would you have the parameters for OMA-URI ?

**Reply by ennnbeee** [reply]
I dont have the User Rights settings in OMA-URI format, as it was far easier to do it via a Settings Catalog profile, than have to create a custom profile and use https://learn.microsoft.com/en-us/windows/client-management/mdm/policy-csp-userrights as a reference for configuring each policy.

I'm glad this post helped, I would recommend using Settings Catalog where possible, unless you really have to use applicability rules.

**Reply by alevalesantos** [reply]
Yes I understand.
But using OMA-URI I have a only one policy with all CIS items and using the Settings Catalog it is necessary to create separate policies.
btw, I appreciate the support and the article, I will continue the tests, everthing working as expected I will reply to you, thanks

**Reply by ennnbeee** [reply]
Ahhhh that makes more sense, if I find time, I'll see if I can convert the existing policies to Custom profiles, I do wonder what the item limit is for a Custom Profile though.

**Comment by Wizkeezy** [comment]
This content is amazing, I have been struggling with all these things for a year.

Thank you.

**Comment by jimmywinberg** [comment]
There seems to be an issue with: CIS (L1) User Rights Global - Windows 11 Intune 3.0.1
You are using: (<![CDATA[]]>) this results in error in the Event viewer: Event ID: 821 and 806, Catastrophic failure.
It will however set it to blank on the device.

Changing: (<![CDATA[]]>) to just "" will resolve this issue, no more error in event viewer and policy is applied correctly.

**Reply by jimmywinberg** [reply]
Forgott to encoding "<![CDATA[]]>" to ""

**Reply by jimmywinberg** [reply]
< ![CDATA[]] > to ""

**Reply by ennnbeee** [reply]
Thanks for this, I've updated the exported JSON file.

**Reply by jimmywinberg** [reply]
After testing this on multiple devices etc. it seems as both CDATA and "" will result in the same error during device booting.
Both achieves the same result by effectively setting the UserRight to blank value.

I'm starting to think the only way around this is using the SID: S1-0-0 as this would equal a Null SID.

**Reply by ennnbeee** [reply]
Thanks for the testing mate, it's massively appreciated. I've got some time today, so will have a look at my test machines and see what I can do to find a solution as well.

**Comment by jimmywinberg** [comment]
The Null sid works perfectly and it will say null sid in secpol also making auditing easier.

I have some warnings on Firewall so will review that later also.
________________________________

**Comment by alevalesantos** [comment]
In my case the < ![CDATA[]] , "" or others methods doesnt work.
For "null/empty", I create a file .xml and added the string/string, select string (XML file) and select the .xml file.
Work as expected

**Reply by ennnbeee** [reply]
I've updated the exported profile and the page with the null SID as per the comment from @jimmywinberg

**Reply by alevalesantos** [reply]
Amazing, I will export my current CIS configuration to upload to the community, to help others : )

**Comment by alevalesantos** [comment]
Attached my current CIS file, have some additional points to fix, I still working, but now are covering 90% of CIS

I used the documentation to export/import the file:
https://github.com/Micke-K/IntuneManagement

[CIS Microsoft Intune for Windows Benchmark v3.0.1 (PRD).json](https://github.com/user-attachments/files/16469689/CIS.Microsoft.Intune.for.Windows.Benchmark.v3.0.1.PRD.json)
